templates/enduro.toml.j2

```diff
@@ -48,6 +48,19 @@ redisChannel = "enduro-events"
 listen = "{{ enduro_internal_api_listen }}"
 debug = "{{enduro_internal_api_debug }}"
 
+[internalStorage.bucket]
+url = "{{enduro_internal_storage_url }}"
+endpoint = "{{enduro_internal_storage_endpoint }}"
+pathStyle = "{{enduro_internal_storage_pathStyle }}"
+accessKey = "{{enduro_internal_storage_key }}"
+secretKey = "{{enduro_internal_storage_secret }}"
+region = "{{enduro_internal_storage_region }}"
+bucket = "{{enduro_internal_storage_bucket }}"
+
+[internalStorage.azure]
```

Comment:
This should be only added when enduro_internal_storage_azure_account is set

Reply:
Ignored configuration options are distracting and misguiding. And in this particular case, naming a cloud provider that the user might not be using at all, is confusing to me.

We have other cases of  conditional values in the same template file if you need references for the change.

Reply:
Done, I've amended the same commit ;)

Reply:
Good point, I've used `{% if enduro_internal_storage_azure_account and enduro_internal_storage_azure_key %}` now. That will check if both variables are defined and not empty.